Logos/Morphisec_Logo.svg

```diff
@@ -0,0 +1,6 @@
+<svg id="f8a78c88-c469-46fa-be85-47fa820f7cd7" viewBox="0 0 150 150">
+<title>
```

Comment:
Please remove title tag and add  xmlns="http://www.w3.org/2000/svg"  in svg  tag. Please  refer file below.
[morphiseccode.txt](https://github.com/Azure/Azure-Sentinel/files/5753765/morphiseccode.txt)

Reply:
@v-admahe  Uploaded new logo with no title and added the xmlns back